src/layouts/hass-error-screen.js

```diff
@@ -28,6 +28,7 @@ class HassErrorScreen extends PolymerElement {
       </app-toolbar>
       <div class="layout vertical center-center">
         <h3>[[error]]</h3>
+        <slot></slot>
```

Comment:
We should probably make the "go back" button the default if no slot is given. You can do that by putting the content inside the slot tag.